data/bingo/functions/game/start/start_game.mcfunction

```diff
@@ -7,8 +7,9 @@
 execute at @e[type=minecraft:area_effect_cloud, tag=bingo.spawn, limit=1] rotated as @s run tp ~ ~ ~
 scoreboard players set $game_state bingo.state 3
 
-tellraw @s {"translate": "bingo.game.start.countdown_completed"}
-tellraw @s [{"translate": "bingo.game.go_to_lobby", "with": [{"translate": "bingo.game.go_to_lobby.click_lobby", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, {"translate": "bingo.game.go_to_lobby.click_spectator", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}]}, "\n", {"translate": "bingo.game.go_to_lobby.hint", "color": "gray", "italic": true, "with": [{"text": "/trigger bingo.lobby", "color": "white", "italic": false, "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.lobby"}}, {"text": "/trigger bingo.spectator", "color": "white", "italic": false, "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.spectator"}}]}]
+tellraw @s [{"translate": "bingo.game.menu.text", "with": [{"translate": "bingo.game.menu.icon", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.menu"}, "hoverEvent":{"action":"show_text","contents":[{"translate":"bingo.game.menu.hover_text"}]}}]}, " ", {"translate": "bingo.game.start.countdown_completed"}]
```

Comment:
I think the `GO` should not be prefixed by `[≡]`
Perhaps the "Loading terrain" message could be though, as you already can use the triggers in the skybox, and adding the burger menu there might make this more apparent, and it is a nice feature for speedrunners, if the spawn sucks.

I think the initial menu and hint should still come after GO though, to have the menu right at the bottom of the chat when the game starts.

Reply:
Moved the icon from "GO!" to the "Loading Terrain..." message.